### PR TITLE
first huevon

### PR DESCRIPTION
**Pull request by maximustxt**
zasdfaefaefefseaf-ñlk<srdgnkjl-sgnmsgdknsdfknlsgfknñlsgdfzklngdfszmnbzdfgkndfszknlfgdklnsfrlknfsdklndfzgknsfdznkzgdf,mgndf<gm,fdgmdfgnmdfgnd<m,fgndm,fgnd

### CODE REVIEW

**Review by JosePergasere — COMMENTED** [review]
No

File: Nuevo documento de texto.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 Gato de mierda 
+
+Perro culia
```
Comment (EzeTorcetta) [inline Nuevo documento de texto.txt:3]: culo roto

File: Nuevo documento de texto.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 Gato de mierda
```
Comment (EzeTorcetta) [inline Nuevo documento de texto.txt:1]: lololo

**Review by EzeTorcetta — APPROVED** [review]
te lo apruebo papi

**Pull request closed without merging**